src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -266,11 +266,23 @@ winrt::hstring Profile::ExpandedIconPath() const
 // - This profile's expanded background image path / the empty string.
 winrt::hstring Profile::ExpandedBackgroundImagePath() const

```

Comment:
Please update the comment above (Method Description).

Reply:
Got ti